### DISCUSSION TITLE
posts/upcoming-decorator-enhancements-in-autofac-4-9

### DISCUSSION THREAD

**Discussion by giscus** [alexmg/blog#3] [Announcements]
# posts/upcoming-decorator-enhancements-in-autofac-4-9

Autofac 4.9 introduces new overloads of the existing decorator registration methods that allow for implicit ordering and conditional decoration of services using registration order, as well as support for open-generic registrations and relationship types.

https://blog-alexmg.vercel.app/posts/upcoming-decorator-enhancements-in-autofac-4-9

**Comment by alexmg** [comment]
Test comment
